README.md

```diff
@@ -89,6 +89,60 @@ Or use the power of Vue.js within a google map like this:
 </script>
 ```
 
+## Custom markers
+
+To implement custom marker, best practice is to create a map settings file, which you can import into the page(s) where you use the gmap-map and gmap-marker tags. This is usually done in a project configuration folder.
+
+An example of the map settings file:
+
+**mapSettings.js**
+
+```javascript
+import { loaded } from 'vue2-google-maps';
+
+var settings = new Vue({
+    data: {
+        defaultIconSettings: {},
+        defaultMapOptions: {
+          /* below, useful default options */
+          clickableIcons: false,
+          streetViewControl: false,
+          panControlOptions: false,
+          gestureHandling: 'greedy'
+        },
+    }
+})
+export default settings;
+
+loaded.then(() => {
+    // (1) see note below for information about what is available from
+    // the google.maps object.
+    settings.defaultIconSettings = {};
```

Comment:
I think it is worth showing how `google.maps` can be used here, e.g. for `google.maps.Size`.